template/src/features/counter/counterSlice.js

```diff
@@ -8,7 +8,7 @@ export const slice = createSlice({
   reducers: {
     increment: state => {
       // Redux Toolkit allows us to write "mutating" logic in reducers. It
-      // doesn't actually mutate the state because it uses the immer library,
+      // doesn't actually mutate the state because it uses the inner library,
```

Comment:
How about we capitalize it instead, so it's more obvious that it's a proper noun naming the library
```suggestion
      // doesn't actually mutate the state because it uses the Immer library,
```